src/test/java/org/openstreetmap/atlas/tags/HighwayTagTestCase.java

```diff
@@ -0,0 +1,44 @@
+package org.openstreetmap.atlas.tags;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openstreetmap.atlas.tags.annotations.validation.Validators;
+import org.openstreetmap.atlas.utilities.testing.TestTaggable;
+
+/**
+ * Test for {@link org.openstreetmap.atlas.tags.HighwayTag}
+ *
+ * @author v-garei
+ */
+public class HighwayTagTestCase
+{
+
+    @Test
+    public void highwayTagTests()
+    {
+        final Taggable emergencyBayTaggable = Taggable.with(HighwayTag.KEY,
+                HighwayTag.EMERGENCY_BAY.toString());
+        Assert.assertTrue(Validators.hasValuesFor(emergencyBayTaggable, HighwayTag.class));
+
+        final TestTaggable falseNodeOnlyTag = new TestTaggable(HighwayTag.KEY,
+                HighwayTag.TRUNK.toString());
+        Assert.assertFalse(HighwayTag.isNodeOnlyTag(falseNodeOnlyTag));
+
+        final TestTaggable falseWayOnlyTag = new TestTaggable(HighwayTag.KEY,
+                HighwayTag.BUS_STOP.toString());
+        Assert.assertFalse(HighwayTag.isWayOnlyTag(falseWayOnlyTag));
+
+        final TestTaggable trueNodeOnlyTag = new TestTaggable(HighwayTag.KEY, "traffic_mirror");
```

Comment:
Did you want to also change this one and the two below to use the enum?

Reply:
> Did you want to also change this one and the two below to use the enum?

yeah I am dumb, for some reason I was thinking that since these ones were new they wouldn't be accessible yet.. whoops